templates/part.board.mainView.php

```diff
@@ -5,36 +5,39 @@
 		<p>{{ statusservice.text }}</p></div>
 </div>
 <div id="board-header">
-	<a href="#" title="<?php p($l->t('All Boards')); ?>">
-		<i class="icon icon-home"></i>
-	</a>
+	<div id="home" class="crumb svg last">
```

Comment:
That icon is a decorator of the "Home" button (which, by the way, should probably be renamed to "All boards") so it should be part of the CSS, not HTML. `<img>` is for content, `background-image` is for design. So Files app is doing it wrong.

But there is one more thing to consider - accessibility. From that point of view, `title` tag should suffice, but to be 100% compatible with screen readers, we might want to add a screen-reader-only text inside `<a>` tag. Botsrap is using this class for that purpose:
```
.sr-only {
  position: absolute;
  width: 1px;
  height: 1px;
  padding: 0;
  margin: -1px;
  overflow: hidden;
  clip: rect(0, 0, 0, 0);
  border: 0;
}
.sr-only-focusable:active,
.sr-only-focusable:focus {
  position: static;
  width: auto;
  height: auto;
  margin: 0;
  overflow: visible;
  clip: auto;
}
```